app/src/shared/modules/user/actions.js

```diff
@@ -151,6 +151,19 @@ export const updateCurrentUserTimezone = (timezone: string) => (dispatch: Functi
     }))
 }
 
+export const updateCurrentUserImage = (image: ?string) => (dispatch: Function, getState: Function) => {
+    const user = selectUserData(getState())
+    return services.uploadProfileAvatar()
+        .then(() => {
+            dispatch(updateCurrentUser({
+                ...user,
+                imageUrl: image,
+            }))
+        }, (error) => {
+            throw error
```

Comment:
Looks redundant. If you skip the error callback the uncaught remains uncaught, right? 😉